server/controllers/users.js

```diff
@@ -0,0 +1,9 @@
+const models = require('../models');
+
+module.exports = {
+
+  get: /* models.users.getAll */ (res, req) => {},
+
+  post: /* models.users.create */ (res, req) => {},
+
+}
```

Comment:
This is a nitpick so please feel free to resolve it right away, but this red symbol is because git and text editors like a newline at the end of the file.

Apparently it can cause reading issues for some text editors.